src/PrestaShopBundle/Routing/Converter/CacheProvider.php

```diff
@@ -110,24 +131,8 @@ private function serializeLegacyRoutes(array $legacyRoutes)
         return json_encode($flattenRoutes);
     }
 
-    /**
-     * @param string $serializedLegacyRoutes
-     *
-     * @return LegacyRoute[]
-     */
-    private function unserializeLegacyRoutes($serializedLegacyRoutes)
+    public function clearCache(): void
```

Comment:
Can you please keep the public functions at the top of the class? we respect Symfony convention for this (or at least e try to 😅)